src/bin/setupScripts.js

```diff
@@ -1,18 +1,39 @@
-const readFile = require("../utils/readFile");
-const createFile = require("../utils/createFile");
-const chalk = require("chalk")
+import readFile from "../utils/readFile.js";
+import createFile from "../utils/createFile.js"; 
+
+import chalk from "chalk";
 
 /**
  * Reads the package.json file in the project directory,
  * updates the 'scripts' field with the necessary scripts for running
  * the server using nodemon, and writes the updated package.json file.
  * 
- * @param {string} projectName The name of the project to setup scripts for
+ * @param {string} projectName The name of the project to setup scripts for.
  */
+
+export const setupScripts = async (projectName) => {
```

Comment:
this is a repeated export. please remove this. also please make sure you update the comment too if requrired.